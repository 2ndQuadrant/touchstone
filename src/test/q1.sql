:s
SELECT :1;
:e
