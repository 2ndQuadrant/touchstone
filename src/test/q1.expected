-- Q 1
-- 1: ONE
BEGIN;
SELECT ONE;
COMMIT;
